pymeasure/instruments/keysight/keysight34980A.py

```diff
@@ -0,0 +1,628 @@
+        dc_voltage = str(temp_values[0])
+        return(dc_voltage)
+
+    def measure_ac_voltage(self, chan, resolution = "DEFault", range_param = "AUTO"):    
+        """
+        The following command configures the instrument for ac voltage measurements, 
+        triggers the internal DMM to scan the specified channel.
+        """
+        temp_values = self.values(":MEASure:SCALar:VOLTage:AC?")
+        return(str(temp_values[0]))
+
+    def measure_dc_current(self, chan, resolution = "DEFault", range_param = "AUTO"):    
+        """
+        The following command configures the instrument for dc current measurements, 
+        triggers the internal DMM to scan the specified channel.
+        """
+        temp_values = self.values(":MEASure:SCALar:CURRent:DC?")
+        return(str(temp_values[0]))
+
+    def measure_ac_current(self, chan, resolution = "DEFault", range_param = "AUTO"):
+        """
+        The following command configures the instrument for ac voltage measurements, 
+        triggers the internal DMM to scan the specified channel.
+        """
+        temp_values = self.values(":MEASure:SCALar:CURRent:AC?")
+        return(str(temp_values[0]))
+
+    def measure_frequency(self, chan, resolution = "DEFault", range_param = "DEFault"):    
+        """
+        The following command configures the instrument for frequency measurements, 
+        triggers the internal DMM to scan the specified channel.
+        """
+        temp_values = self.values(":MEASure:SCALar:FREQuency?")
+        return(str(temp_values[0]))
+
+    def measure_period(self, chan, resolution = "DEFault", range_param = "DEFault"):    
+        """
+        The following command configures the instrument for period measurements, 
+        triggers the internal DMM to scan the specified channel.    
+        """
+        temp_values = self.values(":MEASure:SCALar:PERiod?")
+        return(str(temp_values[0]))
+
+    def measure_resistance(self, chan, resolution = "DEFault", range_param = "AUTO"):    
+        """
+        The following command configures the instrument for resistance measurements, 
+        triggers the internal DMM to scan the specified channel.
+        """
+        temp_values = self.values(":MEASure:SCALar:RESistance?")
+        return(str(temp_values[0]))
+
+    def measure_temperature(self, chan, probe = "TCouple", probe_type = "T", resolution = "DEFault", range_param = "AUTO"):   
+        """
+        The following command configures the instrument for temperature measurements on the specified channel.
+        """
+        temp_values = self.values(":MEASure:SCALar:TEMPerature?")
+        return(str(temp_values[0]))
+
+    def set_temperature_unit(self, chan = "", unit = 'K'):    
+        """
+        This command selects the temperature units (°C, °F, or Kelvins) on the specified channels. 
+        If you omit the optional <ch_list> parameter, this command applies to the internal DMM, 
```

Comment:
for methods with arguments it is always nice to have at the end of the docstring a list of the parameters in the following format:
```
:param chan: A optional string parameter that defines the channel for which the temperature unit is set.
:param unit: An optional string parameter that defines which unit to set.
```